examples/PySDM_examples/deJong_Azimi/simulation1D.py

```diff
@@ -0,0 +1,227 @@
+from collections import namedtuple
+
+import numpy as np
+from PySDM_examples.Shipway_and_Hill_2012.mpdata_1d import MPDATA_1D
```

Comment:
would it now make sense to move the `MPDATA_1D` class to a common folder, like `examples/utils`?

Reply:
Hmm, I think that's a choice that I would prefer to leave up to you. My inclination is to keep it within the canonical example that it is specific to.

Reply:
In fact it would be best to reduce code duplication between `Shipway_and_Hill_2012` and `deJong_Azimi` so that if someone fixes something in the S&H simulation code, we see it here as well. Why not inherit from S&H Simulation class?